lib/active_record_doctor/detectors/missing_presence_validation.rb

```diff
@@ -79,6 +79,11 @@ def presence_validator_present?(model, column)
             (validator.attributes & allowed_attributes).present?
         end
       end
+
+      def inclusion_validator_items(validator)
+        items = validator.options[:in] || validator.options[:within] || []
+        items.is_a?(Enumerable) ? items : []
```

Comment:
It seems that a `Proc` validator is equivalent to `in: []`. I don't think that's a good choice for users, as `active_record_doctor` shouldn't be reporting errors indiscriminately.

For now, I suggest _skipping_ those validators that use a `Proc`.